vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -354,6 +390,45 @@ void WinRTWebSocketResource::Synchronize()
   }
 }
 
+string WinRTWebSocketResource::GetRestrictedErrorMessage() noexcept
+{
+  string result;
+  CComPtr<IRestrictedErrorInfo> errorInfo;
+  if (SUCCEEDED(GetRestrictedErrorInfo(&errorInfo)))
+  {
+    BSTR description;
+    HRESULT error;
+    BSTR restrictedDesc;
+    BSTR capabilitySid;
+    if (SUCCEEDED(errorInfo->GetErrorDetails(&description, &error, &restrictedDesc, &capabilitySid)))
+    {
+      try
+      {
+        result = Utf16ToUtf8(description) + ": " + Utf16ToUtf8(restrictedDesc);
+      }
+      catch (const std::exception& e)
+      {
+        result = e.what();
+      }
+    }
+    else
+    {
+      result = "Unknown error";
```

Comment:
unify with line 426. for example, have a scopeExit-type object.

Reply:
This method has been removed.